google-cloud-spanner/src/main/java/com/google/cloud/spanner/BackupInfo.java

```diff
@@ -96,6 +111,8 @@
     private BackupEncryptionConfig encryptionConfig;
     private EncryptionInfo encryptionInfo;
     private com.google.spanner.admin.database.v1.Backup proto;
+    private Timestamp maxExpireTime;
+    private ProtocolStringList referencingBackup;
```

Comment:
We should not use the protobuf specific types here. Instead of `ProtocolStringList`, could you use `List<String>`?